src/raylib.h

```diff
@@ -1150,7 +1152,7 @@ RLAPI float GetGesturePinchAngle(void);                 // Get gesture pinch ang
 //------------------------------------------------------------------------------------
 // Camera System Functions (Module: rcamera)
 //------------------------------------------------------------------------------------
-RLAPI void SetCameraMode(Camera camera, int mode);      // Set camera mode (multiple camera modes available)
+RLAPI void SetCameraMode(Camera* camera, int mode);      // Set camera mode (multiple camera modes available)
```

Comment:
As commented, `SetCameraMode()` can be just removed and its functionality moved to `UpdateCamera(&camera, mode)`.

Reply:
Will be done. :)